api/tasks.go

```diff
@@ -791,21 +791,44 @@ func (wc *WaitConfig) Copy() *WaitConfig {
 	return nwc
 }
 
+type ChangeScriptConfig struct {
+	Path        *string        `mapstructure:"path" hcl:"path"`
+	Args        *[]string      `mapstructure:"args" hcl:"args,optional"`
```

Comment:
Curious why we are using a pointer to an array here?

Reply:
so my thinking was—I wanted to make a `nil` check, to see if the value's set. but of course in go I can make a nil check against slice. sorry it's been a while since I wrote some go, thanks for spotting. 

Fixed.